extensions.md

```diff
@@ -17,15 +17,38 @@ Domain: `opendsr.mparticle.com`
 Supported keys:
 
 - `mpids`: An array of mParticle IDs. The mParticle ID is a 64-bit signed integer.
+- `identities`: An array of additional mParticle-supported identity objects, each containing a key and value. Valid keys are:
```

Comment:
I think you should specify that each value must be a string

Reply:
Updated